core/internal/sdata/schema.go

```diff
@@ -70,8 +71,9 @@ type DBRel struct {
 
 // Add at the top with other type definitions
 type Config struct {
-	AllowedSchemas []string // List of allowed schemas
-	DefaultSchema  string   // Default schema to use
+	AllowedSchemas  []string // List of allowed schemas
+	DefaultSchema   string   // Default schema to use
+	EnableCamelCase bool     // Enable camel case for cross schema names
```

Comment:
I'm trying to decide if this the addition of this level of configuration is necessary. I think people would rather just pass the default separator here instead of wondering if its camelcase or not